package com.activity.todo.exception;

public class TaskForUserNotFoundException extends RuntimeException{

	public TaskForUserNotFoundException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public TaskForUserNotFoundException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public TaskForUserNotFoundException(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public TaskForUserNotFoundException(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public TaskForUserNotFoundException(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}

}
